### PR TITLE
Sep translation and enhancements

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
In Progress

## Description
Symantec Endpoint Protection Manager Version 2

## Does it break backward compatibility?
   - No: New version of the old integration.

## Related issues
https://github.com/demisto/etc/issues/13935

## Must have
- [x] Tests
- [x] Documentation (https://github.com/demisto/etc/issues/14550 - First draft of documentation)
- [x] Code Review

### CODE REVIEW

File: Integrations/integration-SymantecEndpointProtection_V2.yml [hunk 1]
```diff
@@ -0,0 +1,1136 @@
...
+            'HumanReadable': md,
+            'EntryContext': {}
+        })
+
+
+    def list_locations_command(token):
+        group_id = demisto.getArg('groupID')
+        url = 'sepm/api/v1/groups/{}/locations'.format(group_id)
+        url_resp = do_get(token, False, url)
+        location_ids = map(lambda location_string: {'ID': location_string.split('/')[-1]}, url_resp)
```
Comment (yuvalbenshalom) [inline Integrations/integration-SymantecEndpointProtection_V2.yml]: separate business logic functions

File: Integrations/integration-SymantecEndpointProtection_V2.yml [hunk 2]
```diff
@@ -0,0 +1,1136 @@
...
+        action = demisto.getArg('actionType')
+        action_type = 'Quarantine' if action == 'Add' else 'Undo'
+
+        if not endpoint:
+            return_error('Please provide the IP address or the hostname of endpoint.')
+        elif validate_ip(endpoint):
+            computer_id = get_computer_id(token, endpoint, None)
+        else:
+            computer_id = get_computer_id(token, None, endpoint)
+        command_id = quarantine(token, computer_id, action_type)
```
Comment (yuvalbenshalom) [inline Integrations/integration-SymantecEndpointProtection_V2.yml]: i think lines 979-987 should be in business logic functions

File: Integrations/integration-SymantecEndpointProtection_V2.yml [hunk 3]
```diff
@@ -0,0 +1,1136 @@
...
+        endpoint = demisto.getArg('endpoint')
+        scan_type = demisto.getArg('scanType')
+
+        if not endpoint:
+            return_error('Please provide the IP address or the hostname of endpoint.')
+        elif validate_ip(endpoint):
+            computer_id = get_computer_id(token, endpoint, None)
+        else:
+            computer_id = get_computer_id(token, None, endpoint)
+        command_id = scan(token, computer_id, scan_type)
```
Comment (yuvalbenshalom) [inline Integrations/integration-SymantecEndpointProtection_V2.yml]: lines 1012-1018 in separate function

File: Integrations/integration-SymantecEndpointProtection_V2.yml [hunk 4]
```diff
@@ -0,0 +1,1136 @@
...
+        body = [{
+            'group': {
+                'id': group_id
+            },
+            'hardwareKey': hardware_key
+        }]
+        response = do_patch(token, 'sepm/api/v1/computers', body)
+        message = '### Moved client to requested group successfully' \
+            if response[0].get('responseCode') == '200' \
+            else '### Error moving client'
```
Comment (yuvalbenshalom) [inline Integrations/integration-SymantecEndpointProtection_V2.yml]: lines 1068-1077 in separate function

File: Integrations/integration-SymantecEndpointProtection_V2.yml [hunk 5]
```diff
@@ -0,0 +1,1136 @@
...
+    def update_endpoint_content_command(token):
+        endpoint = demisto.getArg('endpoint')
+        if not endpoint:
+            return_error('Please provide the IP address or the hostname of endpoint.')
+        elif validate_ip(endpoint):
+            computer_id = get_computer_id(token, endpoint, None)
+        else:
+            computer_id = get_computer_id(token, None, endpoint)
+
+        command_id = update_content(token, computer_id)
```
Comment (yuvalbenshalom) [inline Integrations/integration-SymantecEndpointProtection_V2.yml]: lines 1040-1047 in separate function

File: Integrations/integration-SymantecEndpointProtection.yml [hunk 6]
```diff
@@ -1,49 +1,454 @@
+category: Endpoint
 commonfields:
-  id: Symantec Endpoint Protection
+  id: Symantec Endpoint Protection V2
```
[the remark this answers is not in the document]
Comment (ShahafBenYakir) [reply Integrations/integration-SymantecEndpointProtection.yml]: This is outdated

File: Integrations/integration-SymantecEndpointProtection_V2.yml [hunk 7]
```diff
@@ -0,0 +1,1136 @@
...
+  required: false
+  type: 0
+- defaultvalue: 'false'
+  display: Trust any certificate (unsecure)
+  name: insecure
+  options: [
+    ]
+  required: false
+  type: 8
+- defaultvalue: 'true'
```
[the remark this answers is not in the document]
Comment (ShahafBenYakir) [reply Integrations/integration-SymantecEndpointProtection_V2.yml]: - [ ]

File: Integrations/integration-SymantecEndpointProtection_V2.yml [hunk 8]
```diff
@@ -0,0 +1,1136 @@
...
+      description: The columns to show.
+      isArray: false
+      name: columns
+      predefined: [
+        ]
+      required: false
+      secret: false
+    description: Get information about groups.
+    execution: false
+    name: sep-groups-info
```
[the remark this answers is not in the document]
Comment (ShahafBenYakir) [reply Integrations/integration-SymantecEndpointProtection_V2.yml]: - [ ] Status

File: Integrations/integration-SymantecEndpointProtection_V2.yml [hunk 9]
```diff
@@ -0,0 +1,1136 @@
...
+            'version': version_json,
+            'avdef': avdef_json
+        }
+
+        md = '## System Information\n'
+        md += tableToMarkdown('Version', version_json)
+        md += tableToMarkdown('AV Definitions', avdef_json)
+
+        context = avdef_json.get('publishedBySymantec')
+        if type(context) is dict    :
```
[the remark this answers is not in the document]
Comment (ShahafBenYakir) [reply Integrations/integration-SymantecEndpointProtection_V2.yml]: - [ ] Status

File: Integrations/integration-SymantecEndpointProtection_V2.yml [hunk 10]
```diff
@@ -0,0 +1,1136 @@
...
+
+
+    def client_content_command(token):
+        time_zone = demisto.getParam('timeZone')
+        client_content_json = do_get(token, False, 'sepm/api/v1/stats/client/content')
+        epoch_time = client_content_json.get('lastUpdated')
+        if time_zone:
+            epoch_time = parse_epoch_to_local(epoch_time, time_zone)
+        last_update_date = timestamp_to_datestring(epoch_time, '%a %b %d %y %H:%M:%S %z')
+        client_version = client_content_json.get('clientDefStatusList')
```
[the remark this answers is not in the document]
Comment (ShahafBenYakir) [reply Integrations/integration-SymantecEndpointProtection_V2.yml]: - [ ] Status

File: Integrations/integration-SymantecEndpointProtection_V2.yml (hunk 1 above)
Comment (ShahafBenYakir) [reply Integrations/integration-SymantecEndpointProtection_V2.yml]: - [ ] Status

File: Integrations/integration-SymantecEndpointProtection_V2.yml (hunk 2 above)
Comment (ShahafBenYakir) [reply Integrations/integration-SymantecEndpointProtection_V2.yml]: - [ ] Status

File: Integrations/integration-SymantecEndpointProtection_V2.yml (hunk 3 above)
Comment (ShahafBenYakir) [reply Integrations/integration-SymantecEndpointProtection_V2.yml]: - [ ] Status

File: Integrations/integration-SymantecEndpointProtection_V2.yml (hunk 5 above)
Comment (ShahafBenYakir) [reply Integrations/integration-SymantecEndpointProtection_V2.yml]: - [ ] Status

File: Integrations/integration-SymantecEndpointProtection_V2.yml (hunk 4 above)
Comment (ShahafBenYakir) [reply Integrations/integration-SymantecEndpointProtection_V2.yml]: - [ ] Status

**Pull request merged**